ruby-gem/calabash-android.gemspec

```diff
@@ -21,7 +21,7 @@ Gem::Specification.new do |s|
   s.add_dependency( "slowhandcuke", '~> 0.0.3')
   s.add_dependency( "rubyzip", "~> 1.1" )
   s.add_dependency( "awesome_print", '~> 1.2.0')
-  s.add_dependency( 'httpclient', '~> 2.3.2')
+  s.add_dependency( 'httpclient', '>= 2.3.2', '< 3.0')
```

Comment:
@krukow Is there any reason not to use:  '~> 2.6' ?

If 2.3 is an acceptable lower limit, then we could use '~> 2.3' here which would be equivalent to:  '>= 2.3, < 3.0'

I don't want to be in a position where we get far behind the current version, like we did with rubyzip.